### PR TITLE
Text::to_bytes

### PR DESCRIPTION
**Pull request by R4PaSs**
Simple addition to the Bytes module, to build one from any Text type

### CODE REVIEW

File: lib/standard/bytes.nit [hunk 1]
```diff
@@ -46,6 +46,27 @@ class Bytes
 		init(ns, 0, 0)
 	end
 
+	# Create a new `Bytes` instance from a String's bytes
+	#
+	# ~~~nit
+	# assert (new Bytes.from_text("String")) == [83u8, 116u8, 114u8, 105u8, 110u8, 103u8]
```
Comment (Morriar) [inline lib/standard/bytes.nit]: Are those parenthesis really mandatory?

**Comment by Morriar** [comment]
+1

File: lib/standard/bytes.nit (hunk 1 above)
Comment (privat) [inline lib/standard/bytes.nit]: isn't a `String::to_bytes` more easier to use?

~~~nit
assert "String".to_bytes isa Bytes
assert "String".to_bytes == [83u8, 116u8, 114u8, 105u8, 110u8, 103u8]
~~~

**Comment by R4PaSs** [comment]
Reroll, fixed both @privat and @Morriar's remarks

**Comment by Morriar** [comment]
+1

**Comment by privat** [comment]
meh. why not the following?

remove the named constructor completely
provide Text::to_bytes with the basic implementation without the `isa`
redefine a faster implem in FlatText if you like (the isa part).

**Comment by R4PaSs** [comment]
Done with @privat's method

**Comment by privat** [comment]
simple and neat. +1

**Pull request merged**